deploy/hubble/manifests/controller/helm/retina/templates/agent/daemonset.yaml

```diff
@@ -196,7 +196,11 @@ spec:
           command:
             - powershell.exe
             - -command
+            {{- if semverCompare ">=1.30" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
I guess it makes sense since it runs in/against a cluster, but to confirm: Helm knows the K8s version and we can just get it like this?

Reply:
from my testing it got assigned the correct command according to the k8s version. I referenced [peer-service.yaml](https://github.com/microsoft/retina/blob/main/deploy/hubble/manifests/controller/helm/retina/templates/hubble/peer-service.yaml#L27) when making this change.